source/extensions/filters/network/thrift_proxy/config.h

```diff
@@ -96,6 +98,8 @@ class ConfigImpl : public Config,
 
   std::list<ThriftFilters::FilterFactoryCb> filter_factories_;
   const bool payload_passthrough_;
+
+  uint64_t max_requests_per_connection_ = UINT64_MAX;
```

Comment:
nit:

```
uint64_t max_requests_per_connection_{};
```

initializing this to zero and skipping the limit check when zero is probably easier (and more common across the code base).

Reply:
Get it.